setup/terminal.md

```diff
@@ -0,0 +1,33 @@
+# Why terminal 
+
+Majority of people when they first start to use the computers they do it through the GUI (Graphical user interface). They do it by clicking or touching things on the screen. 
+This is the best way of interacting with computers today, maybe in near future we will use voice more and more, however this is not always the fastest way of doing things on the computer.
+
+For programmers the best way is to do things from terminal level. 
+There is couple of reasons for it, for example:
+
+* common operations on files is much faster when you do it from terminal, as quite often few commands on the terminal can do as much as thousand clicks on the screen
```

Comment:
common operations on files are much faster when you do them from the terminal, (as quite often few commands on the terminal can do as much as thousand clicks on the screen)
the part in the brackets is hard to understand, try to explain what you were trying to say better